api-helpers/api-hofs.ts

```diff
@@ -64,21 +66,24 @@ export const withAsync = (
         );
         return res.status(500).end();
       }
+
       if (result === null) {
         return res.status(204).end();
       }
+
       return res.json(result);
     } catch (err) {
       if (Boom.isBoom(err)) {
         Object.entries(err.output.headers).forEach(([key, val]) => res.setHeader(key, val));
         return res.status(err.output.statusCode).json(err.output.payload);
       } else {
         logger.error(err);
+        /* eslint-disable */
         Sentry.captureException(err);
```

Comment:
Tu nie brakuje `await`?

Reply:
Dlaczego? Myślę, że może to się wykonać asynchronicznie, po co blokować wątek czymś co jest jednak bardziej side effectem.

Reply:
W Vercelu nie można tak robić

Reply:
Sorry, to nie jest Promise w ogóle ;P